
void foo()
{
	loop {
		redo;
	}
}

========

negative/LoopStm.syp:5: redo invalid outside a loop/switch block
found 1 errors
